docs/4.1/trustedclusters.md

```diff
@@ -30,12 +30,12 @@ This guide's focus is on more in-depth coverage of trusted clusters features and
 !!! tip "Teleport Node Tunneling":
     If you have a large amount of devices on different networks, such as
     managed IoT devices or a couple of nodes on a different network you can utilize
-    the [Teleport Node Tunneling](quickstart.md#adding-a-node-located-behind-nat-teleport-node-tunneling).
+    the [Teleport Node Tunneling](admin-guide.md#adding-a-node-located-behind-nat-teleport-node-tunneling).
 
 
 ## Introduction
 
-As explained in the [architecture document](architecture/#core-concepts),
+As explained in the [architecture document](teleport_architecture_overview/#design-principles),
```

Comment:
Should be linked to `architecture/teleport_architecture_overview.md#design-principles`